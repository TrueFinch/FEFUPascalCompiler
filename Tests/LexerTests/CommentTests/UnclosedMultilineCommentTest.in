{ops, 
            unclosed multiline comment
       such a failure