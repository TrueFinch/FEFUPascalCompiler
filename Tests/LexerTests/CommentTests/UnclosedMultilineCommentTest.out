(1,1) Unclosed multiline comment lexeme {ops, \n            unclosed multiline comment\n       such a failure